lib/Conversion/TorchToLinalg/TorchToLinalg.cpp

```diff
@@ -2717,6 +2850,41 @@ class ConvertAtenOnesOp : public OpConversionPattern<AtenOnesOp> {
 };
 } // namespace
 
+namespace {
+
+struct ConvertScalarOp : ConversionPattern {
+  ConvertScalarOp(TypeConverter &typeConverter, MLIRContext *context)
+      : ConversionPattern(typeConverter, MatchAnyOpTypeTag(), /*benefit=*/1,
+                          context) {}
+
+  LogicalResult
+  matchAndRewrite(Operation *op, ArrayRef<Value> operands,
+                  ConversionPatternRewriter &rewriter) const override {
+    if (!isa<AtenAddIntOp>(op))
+      return rewriter.notifyMatchFailure(op, "not a supported elementwise op");
+
+    if (failed(verifyLinalgCompatibleTypes(op, rewriter)))
+      return failure();
+
+    Location loc = op->getLoc();
+    auto resultType = getTypeConverter()
+                          ->convertType(op->getResult(0).getType());
+
+    if (auto addIntOp = dyn_cast<AtenAddIntOp>(op)){
```

Comment:
this belongs in lib/Conversion/TorchToStd/TorchToStd.cpp (actually, we should split that now that std was split: https://github.com/llvm/torch-mlir/issues/409 but for the purpose of this PR, TorchToStd is fine.)